code/modules/mob/living/simple_animal/bot/ed209bot.dm

```diff
@@ -418,6 +418,41 @@
 	new /obj/effect/decal/cleanable/blood/oil(loc)
 	..()
 
+///Disassembling the bot in a civilized manner with a multitool
+/mob/living/simple_animal/bot/ed209/disassemble()
+	walk_to(src,0)
```

Comment:
What is this needed for?

Reply:
I removed all `walk_to(src,0)` and `on = FALSE` from the `disassemble()` and noticed no problem while interrupting movement / actions of the bots by disassembling them.